src/gui/widgets/TrackContentWidget.cpp

```diff
@@ -573,6 +581,15 @@ void TrackContentWidget::mousePressEvent( QMouseEvent * me )
 
 
 
+void TrackContentWidget::mouseReleaseEvent( QMouseEvent * me )
+{
+	gui->songEditor()->syncEditMode();
+	QWidget::mousePressEvent(me);
```

Comment:
```suggestion
	QWidget::mouseReleaseEvent(me);
```

I just noticed this, was it intentional or a typo?